.github/workflows/mac.yml

```diff
@@ -192,15 +192,18 @@ jobs:
           cmake --build ${{ env.BUILD_DIR }} --parallel --config ${{ env.CMAKE_BUILD_TYPE }}
 
       - name: CMake configure, build and install - OpenVINO JS API
-        if: ${{ 'false' }} # 128446
-        # if: fromJSON(needs.smart_ci.outputs.affected_components).JS_API
-        run:
-          cmake -DCPACK_GENERATOR=NPM -DENABLE_SYSTEM_TBB=OFF -UTBB* -S ${{ env.OPENVINO_REPO }} -B ${{ env.BUILD_DIR }}
+        if: fromJSON(needs.smart_ci.outputs.affected_components).JS_API
+        run: |
+          cmake \
+            -DCPACK_GENERATOR=NPM \
+            -DENABLE_SYSTEM_TBB=OFF \
+            -DENABLE_INTEL_GPU=OFF \
```

Comment:
let's drop `ENABLE_SYSTEM_TBB` and `ENABLE_INTEL_GPU` for macOS since these options by default have proper values